config/toml.go

```diff
@@ -270,7 +270,7 @@ pprof-laddr = "{{ .RPC.PprofListenAddress }}"
 #######################################################
 [p2p]
 
-# Enable the new p2p layer.
+# Enable the legacy p2p layer.
```

Comment:
Possibly also:

> This setting is only effective if use-p2p is also enabled.

?

Reply:
Ah this is for the p2p config not the state sync config, I was just following up on a typo from a previous PR